### PR TITLE
 Let the CLI command fetch the data from the data repository 

### LINKED ISSUE #121: Script to update language files
I just created https://github.com/punic/data which contains data files with all the CLDR languages, in PHP Punic format.

We now need to get rid of the `punic-build-data` command in the [`bin`](https://github.com/punic/punic/tree/7456190cdd5e10b9c79c184fd9c6dbba436854d3/bin) folder, and add a new command that fetches these data files.

### PR DESCRIPTION
**Pull request by mlocati**
Fix #121

@Remo @c960657 what about this approach?

Adding new languages just requires:
- `allow_url_fopen` INI option set to true (to download remote files)
- run the command (for adding `it_CH` for example) `./bin/punic-data -l it_CH`

### CODE REVIEW

**Comment by c960657** [comment]
This is very cool!

I suggest that data files are written to a local directory containing the CLDR version (31) and Punic format version (1). This will work as a cache buster, so you are sure to get fresh data files when running `bin/punic-data` after having upgraded Punic. Optimally you would just run `bin/punic-data` after `composer install` (possibly in a post-install hook) and not worry about adding `-r` manually.

Also, it might be worth adding a major-minor version to the Punic format version (1.0, 1.1, etc.). New backwards-compatible changes go into the same major version, and the updated files can just be overwrite the existing files on the server, e.g. `/1/31/en-GB`. But on the client, the local directory should include the full version number, e.g. `/1.2/31/en-GB` to ensure that the updated files are downloaded.

File: bin/punic-data [hunk 1]
```diff
@@ -0,0 +1,967 @@
++#!/usr/bin/env php
```
Comment (c960657) [inline bin/punic-data]: Stray `+` here.

Comment (mlocati) [reply bin/punic-data]: > Stray + here.

Fixed, thanks!

**Comment by mlocati** [comment]
> I suggest that data files are written to a local directory containing the CLDR version (31) and Punic format version (1). This will work as a cache buster

If we do that, we should store somewhere the CLDR version, and read it back. I don't know if it's worth: CLDR versions are updated very rarely... (same argument about the major-minor version of the Punic data).

> Optimally you would just run bin/punic-data after composer install (possibly in a post-install hook) and not worry about adding -r manually.

The way we download remote files if very limited. For instance, it does not support defining a proxy, and someone may even have `allow_url_fopen` disabled. Since CLDR is updated rarely, we can simply release a new Punic version when we update the CLDR / Punic data structure...

**Comment by c960657** [comment]
True, simply bumping the Punic version every time the data changes is probably a simpler and more programatic approach.

**Comment by mlocati** [comment]
:+1:

**Pull request merged**